connector/github/github.go

```diff
@@ -586,7 +594,7 @@ func (c *githubConnector) teamsForOrg(ctx context.Context, client *http.Client,
 
 		for _, team := range teams {
 			if team.Org.Login == orgName {
-				groups = append(groups, team.Name)
+				groups = append(groups, c.teamNamesExtractor(team)...)
```

Comment:
"teamNamesExtractor" is overkill. Just use a switch statement :) It's waaay more readable.